pages/add/form/detail-info-fields.js

```diff
@@ -35,7 +64,7 @@ let Tags = React.createClass({
     this.updateTags(tags);
   },
   handleAddition: function(tag) {
-    const tags = [].concat(this.state.tags, tag);
+    const tags = [].concat(this.state.tags, tag.trim());
```

Comment:
Nope, needs to be `tag.name.trim()` on the line before. `tag` is an object, so we can't directly trim it, and we can't pass `concat` just the name string.